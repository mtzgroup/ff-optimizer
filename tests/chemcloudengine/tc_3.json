{"id": null, "input_data": {"id": "3", "schema_name": "qcschema_input", "schema_version": 1, "molecule": {"schema_name": "qcschema_molecule", "schema_version": 2, "validated": true, "symbols": ["O", "H", "H", "O", "H", "H"], "geometry": [1.29257274, -3.2446599, -3.94007921, 0.75022131, -2.11460352, -2.73254395, 2.22609735, -4.12527227, -5.29879189, -1.15273297, 2.01633787, 0.86927408, -0.59715348, 0.3325918, 0.66707331, -1.73854816, 3.71331191, 1.19052744], "name": "H4O2", "molecular_charge": 0.0, "molecular_multiplicity": 1, "fix_com": false, "fix_orientation": false, "provenance": {"creator": "QCElemental", "version": "v0.23.0", "routine": "qcelemental.molparse.from_schema"}}, "driver": "gradient", "model": {"method": "fail", "basis": "6-31gss"}, "keywords": {}, "protocols": {}, "extras": {}, "provenance": {"creator": "QCElemental", "version": "v0.23.0", "routine": "qcelemental.models.results", "cpu": "Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz", "hostname": "1ee1a2d8cf4c", "username": "root", "qcengine_version": "v0.18.0+36.g0490362", "wall_time": 0.004443645477294922}}, "success": false, "error": {"error_type": "unknown_error", "error_message": "QCEngine Execution Error:\nTraceback (most recent call last):\n  File \"/opt/conda/lib/python3.7/site-packages/qcengine/util.py\", line 114, in compute_wrapper\n    yield metadata\n  File \"/opt/conda/lib/python3.7/site-packages/qcengine/compute.py\", line 91, in compute\n    output_data = executor.compute(input_data, config)\n  File \"/opt/conda/lib/python3.7/site-packages/qcengine/programs/terachem_pbs.py\", line 103, in compute\n    return client.compute(input_model)\n  File \"/opt/conda/lib/python3.7/site-packages/tcpb/tcpb.py\", line 152, in compute\n    job_input_msg = atomic_input_to_job_input(atomic_input)\n  File \"/opt/conda/lib/python3.7/site-packages/tcpb/utils.py\", line 48, in atomic_input_to_job_input\n    ji.method = pb.JobInput.MethodType.Value(ai_copy.model.method.upper())\n  File \"/opt/conda/lib/python3.7/site-packages/google/protobuf/internal/enum_type_wrapper.py\", line 76, in Value\n    self._enum_type.name, name))\nValueError: Enum MethodType has no value defined for name 'FAIL'\n", "extras": null}, "extras": null}