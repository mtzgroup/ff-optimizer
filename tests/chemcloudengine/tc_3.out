Startfile from command line: tc_3.in


       ***********************************************************
       *                    TeraChem v1.9-2021.04-dev            *
       *                 Git Version: 495ec1677c88+               *
       *                   Development Version                   *
       *           Chemistry at the Speed of Graphics!           *
       ***********************************************************
       * This program may only be used in connection with        *
       * a valid license from PetaChem, LLC. Use of this program *
       * or results thereof indicates acceptance of all terms    *
       * and conditions stated in the license and that a valid   *
       * license agreement between the user and PetaChem, LLC    *
       * exists. PetaChem, LLC does not warrant the correctness  *
       * of results or their suitability for any purpose.        *
       * Please email bugs, suggestions, and comments to         *
       *                  help@petachem.com                      *
       *                                                         *
       ***********************************************************


       ***********************************************************
       *  Compiled by henryw7      Fri Apr  9 18:30:10 PDT 2021  *
       *  Supported architectures: sm_30 sm_35 sm_52 sm_61 sm_70 *
       *  Cuda compilation tools, release 9.0, V9.0.176          *
       ***********************************************************


 Job started   Mon Nov 29 22:08:44 2021
 On fire-14-03 (available memory: 198764 MB)

######################################### RUNTIME INFO ##########################################
terachem tc_3.in 

NVRM version: NVIDIA UNIX x86_64 Kernel Module  460.32.03  Sun Dec 27 19:00:34 UTC 2020
GCC version:  gcc version 4.8.5 20150623 (Red Hat 4.8.5-28) (GCC) 

	linux-vdso.so.1 =>  (0x00007fff2d9a9000)
	libcurl.so.4 => /usr/lib64/libcurl.so.4 (0x00007faaefdd6000)
	libmpi.so.12 => /global/software/intel/2017.8.262/compilers_and_libraries_2017.8.262/linux/mpi/intel64/lib/libmpi.so.12 (0x00007faaef0a0000)
	libdftbplus.so => /global/software/TeraChem/2021.02-intel-2017.8.262-CUDA-9.0.176/lib/libdftbplus.so (0x00007faaeebb9000)
	libfsockets.so => /global/software/TeraChem/2021.02-intel-2017.8.262-CUDA-9.0.176/lib/libfsockets.so (0x00007faaee9b5000)
	libxmlf90.so => /global/software/TeraChem/2021.02-intel-2017.8.262-CUDA-9.0.176/lib/libxmlf90.so (0x00007faaee792000)
	libthcbox.so.1 => /global/software/TeraChem/2021.02-intel-2017.8.262-CUDA-9.0.176/lib/libthcbox.so.1 (0x00007faaecae6000)
	libOpenMM.so => /global/software/OpenMM/7.4.1-intel-2017.0.098-CUDA-9.0.176/lib/libOpenMM.so (0x00007faaec484000)
	libgpubox.so.1 => /global/software/TeraChem/2021.02-intel-2017.8.262-CUDA-9.0.176/lib/libgpubox.so.1 (0x00007faaec271000)
	libtensorbox.so.1 => /global/software/TeraChem/2021.02-intel-2017.8.262-CUDA-9.0.176/lib/libtensorbox.so.1 (0x00007faaec056000)
	libmoleculebox.so.1 => /global/software/TeraChem/2021.02-intel-2017.8.262-CUDA-9.0.176/lib/libmoleculebox.so.1 (0x00007faaebe44000)
	libintbox.so.1 => /global/software/TeraChem/2021.02-intel-2017.8.262-CUDA-9.0.176/lib/libintbox.so.1 (0x00007faac776a000)
	libsqmbox.so.1 => /global/software/TeraChem/2021.02-intel-2017.8.262-CUDA-9.0.176/lib/libsqmbox.so.1 (0x00007faac7231000)
	libgridbox.so.1 => /global/software/TeraChem/2021.02-intel-2017.8.262-CUDA-9.0.176/lib/libgridbox.so.1 (0x00007faac6cd4000)
	libdftbox.so.1 => /global/software/TeraChem/2021.02-intel-2017.8.262-CUDA-9.0.176/lib/libdftbox.so.1 (0x00007faac69f2000)
	libcibox.so.1 => /global/software/TeraChem/2021.02-intel-2017.8.262-CUDA-9.0.176/lib/libcibox.so.1 (0x00007faac62db000)
	libccbox.so.1 => /global/software/TeraChem/2021.02-intel-2017.8.262-CUDA-9.0.176/lib/libccbox.so.1 (0x00007faac5e7e000)
	libsimbox.so.1 => /global/software/TeraChem/2021.02-intel-2017.8.262-CUDA-9.0.176/lib/libsimbox.so.1 (0x00007faac5c63000)
	liboptbox.so.1 => /global/software/TeraChem/2021.02-intel-2017.8.262-CUDA-9.0.176/lib/liboptbox.so.1 (0x00007faac5a27000)
	libsandboxbox.so.1 => /global/software/TeraChem/2021.02-intel-2017.8.262-CUDA-9.0.176/lib/libsandboxbox.so.1 (0x00007faac5816000)
	libprotobuf.so.25 => /global/software/protobuf/3.14.0/lib/libprotobuf.so.25 (0x00007faac5363000)
	libcommbox.so.1 => /global/software/TeraChem/2021.02-intel-2017.8.262-CUDA-9.0.176/lib/libcommbox.so.1 (0x00007faac510c000)
	libcuda.so.1 => /usr/lib64/libcuda.so.1 (0x00007faac3a23000)
	libcudart.so.9.0 => /global/software/TeraChem/2021.02-intel-2017.8.262-CUDA-9.0.176/lib/libcudart.so.9.0 (0x00007faac37b6000)
	libcublas.so.9.0 => /global/software/TeraChem/2021.02-intel-2017.8.262-CUDA-9.0.176/lib/libcublas.so.9.0 (0x00007faac0380000)
	libcufft.so.9.0 => /global/software/TeraChem/2021.02-intel-2017.8.262-CUDA-9.0.176/lib/libcufft.so.9.0 (0x00007faab82de000)
	libcusparse.so.9.0 => /global/software/TeraChem/2021.02-intel-2017.8.262-CUDA-9.0.176/lib/libcusparse.so.9.0 (0x00007faab4b78000)
	libcrypto.so.10 => /usr/lib64/libcrypto.so.10 (0x00007faab4717000)
	libssl.so.10 => /usr/lib64/libssl.so.10 (0x00007faab44a4000)
	libstdc++.so.6 => /data/global/software/GCC/4.8.1/lib64/libstdc++.so.6 (0x00007faab41a2000)
	libm.so.6 => /usr/lib64/libm.so.6 (0x00007faab3ea0000)
	libcilkrts.so.5 => /global/software/TeraChem/2021.02-intel-2017.8.262-CUDA-9.0.176/lib/libcilkrts.so.5 (0x00007faab3c64000)
	libgcc_s.so.1 => /data/global/software/GCC/4.8.1/lib64/libgcc_s.so.1 (0x00007faab3a4e000)
	libpthread.so.0 => /usr/lib64/libpthread.so.0 (0x00007faab3832000)
	libc.so.6 => /usr/lib64/libc.so.6 (0x00007faab3464000)
	/lib64/ld-linux-x86-64.so.2 (0x00007faaf0054000)
	libdl.so.2 => /usr/lib64/libdl.so.2 (0x00007faab3260000)
	libidn.so.11 => /usr/lib64/libidn.so.11 (0x00007faab302d000)
	libssh2.so.1 => /usr/lib64/libssh2.so.1 (0x00007faab2e02000)
	libssl3.so => /usr/lib64/libssl3.so (0x00007faab2bb0000)
	libsmime3.so => /usr/lib64/libsmime3.so (0x00007faab2989000)
	libnss3.so => /usr/lib64/libnss3.so (0x00007faab265b000)
	libnssutil3.so => /usr/lib64/libnssutil3.so (0x00007faab242c000)
	libplds4.so => /usr/lib64/libplds4.so (0x00007faab2228000)
	libplc4.so => /usr/lib64/libplc4.so (0x00007faab2022000)
	libnspr4.so => /usr/lib64/libnspr4.so (0x00007faab1de4000)
	libgssapi_krb5.so.2 => /usr/lib64/libgssapi_krb5.so.2 (0x00007faab1b97000)
	libkrb5.so.3 => /usr/lib64/libkrb5.so.3 (0x00007faab18ae000)
	libk5crypto.so.3 => /usr/lib64/libk5crypto.so.3 (0x00007faab167b000)
	libcom_err.so.2 => /usr/lib64/libcom_err.so.2 (0x00007faab1477000)
	liblber-2.4.so.2 => /usr/lib64/liblber-2.4.so.2 (0x00007faab1267000)
	libldap-2.4.so.2 => /usr/lib64/libldap-2.4.so.2 (0x00007faab1012000)
	libz.so.1 => /usr/lib64/libz.so.1 (0x00007faab0dfc000)
	librt.so.1 => /usr/lib64/librt.so.1 (0x00007faab0bf3000)
	libifport.so.5 => /global/software/intel/2017.8.262/compilers_and_libraries_2017.8.262/linux/compiler/lib/intel64/libifport.so.5 (0x00007faab09c4000)
	libifcoremt.so.5 => /global/software/intel/2017.8.262/compilers_and_libraries_2017.8.262/linux/compiler/lib/intel64/libifcoremt.so.5 (0x00007faab0633000)
	libimf.so => /global/software/intel/2017.8.262/compilers_and_libraries_2017.8.262/linux/compiler/lib/intel64/libimf.so (0x00007faab0146000)
	libsvml.so => /global/software/intel/2017.8.262/compilers_and_libraries_2017.8.262/linux/compiler/lib/intel64/libsvml.so (0x00007faaaf228000)
	libiomp5.so => /global/software/intel/2017.8.262/compilers_and_libraries_2017.8.262/linux/compiler/lib/intel64/libiomp5.so (0x00007faaaee83000)
	libintlc.so.5 => /global/software/intel/2017.8.262/compilers_and_libraries_2017.8.262/linux/compiler/lib/intel64/libintlc.so.5 (0x00007faaaec17000)
	libirng.so => /global/software/intel/2017.8.262/compilers_and_libraries_2017.8.262/linux/compiler/lib/intel64/libirng.so (0x00007faaae8a3000)
	libkrb5support.so.0 => /usr/lib64/libkrb5support.so.0 (0x00007faaae694000)
	libkeyutils.so.1 => /usr/lib64/libkeyutils.so.1 (0x00007faaae48f000)
	libresolv.so.2 => /usr/lib64/libresolv.so.2 (0x00007faaae276000)
	libsasl2.so.3 => /usr/lib64/libsasl2.so.3 (0x00007faaae058000)
	libselinux.so.1 => /usr/lib64/libselinux.so.1 (0x00007faaade31000)
	libcrypt.so.1 => /usr/lib64/libcrypt.so.1 (0x00007faaadbf9000)
	libpcre.so.1 => /usr/lib64/libpcre.so.1 (0x00007faaad997000)
	libfreebl3.so => /usr/lib64/libfreebl3.so (0x00007faaad794000)
#################################################################################################

Cannot find license.dat file in the TeraChem installation directory /global/software/TeraChem/2021.02-intel-2017.8.262-CUDA-9.0.176


TRYING THE NETWORK LICENSE...
Connecting to license server '54.208.252.40' port '8877'...
Connected!
Checking your license...

**************************************************************
  Greetings, Martinez Group! You have 1000 licenses in total
  IN USE: 17
  AVAILABLE: 983
**************************************************************

Jobname: 3
Scratch directory: ./scr.3
Random number seed: 745512907

XYZ coordinates 3.pdb
Molden File Output: ./scr.3/3.molden
Using basis set: 6-31gss
dmrgstart not found
Spin multiplicity: 1
DIIS will use up to 10 vectors.
Condition number limit for the DIIS overlap matrix is            0
WF convergence threshold: 3.00e-05
Using DIIS algorithm to converge WF
Maximum number of SCF iterations: 100
Incremental fock with rebuild every 8 iterations
Will switch to conventional Fock if diffuse functions are detected
X-matrix tolerance: 1.00e-04
PRECISION: DYNAMIC
DFT Functional requested: b3lyp
Method: B3LYP  with dispersion correction
  Hartree-Fock exact exchange:          0.20 
  Slater exchange functional:           0.80 
  Becke 1988 exchange functional:       0.72 
  Lee-Yang-Parr correlation functional: 0.81 
  VWN(I) correlation functional:        0.19 
Wavefunction: RESTRICTED

 DFT-D code reimplemented following the
 code by S.Grimme, University Bonn 
 based on DFTD3 V3.2 Rev 0         
 
Please cite DFT-D3 work done with this code as:
S. Grimme, J. Antony, S. Ehrlich and H. Krieg,
J. Chem. Phys. 132 (2010), 154104
If used with BJ-damping cite also
S. Grimme, S. Ehrlich and L. Goerigk,
J. Comput. Chem. 32 (2011), 1456-1465
For DFT-D2 the reference is
S. Grimme, J. Comput. Chem., 27 (2006), 1787-1799
DFT grid type: 1
Using dynamic DFT grids.
Initial guess generated by maximum overlap

********************************************
**** SINGLE POINT GRADIENT CALCULATIONS ****
********************************************
SCF:   Initial guess is taken from previous step

using 2 out of 2 CUDA devices
   Device 0:  GeForce GTX 1080 Ti, 11178MB, CC 6.1 -- CPU THREAD 0
   Device 1:  GeForce GTX 1080 Ti, 11178MB, CC 6.1 -- CPU THREAD 1
-------------------------------------------------------------------
Compiled with MAGMA support.  MAGMA parameters:
    Matrices larger than 5000 square will be treated with MAGMA
       (Change by setting the MagmaMinN environment variable)
    Magma will use 1 out of 2 GPUs
       (Change by setting the MagmaNGPUs environment variable)
-------------------------------------------------------------------
 CPU Memory Available: 96717.88 MegaWords 
 GPU Memory Available: 1141.31 MegaWords 
 Maximum recommended basis set size: 12200 basis functions
 (limited by GPU memory)
-------------------------------------------------------------------
Using d-functions. Configuring GPUs accordingly.
0: CUBLAS initialized, available GPU memory: 8271MB
1: CUBLAS initialized, available GPU memory: 8271MB


Basis set:       6-31gss
Total atoms:     84
Total charge:    0
Total electrons: 316 (158-alpha, 158-beta)
Number electrons modeled by ECPs: 0
Total orbitals:  840
Total AO shells: 378 (210 S-shells; 126 P-shells; 42 D-shells; 0 F-shells; 0 G-shells)
Spin multiplicity: 1
Nuclear repulsion energy (QM atoms): 5416.776587671816 a.u.

DFTD Energy:    -0.1879960 hartree
Setting up the DFT grid...
time to set the grid = 0.04 s
DFT grid points: 78992 (940 points/atom)
Setting up the DFT grid...
time to set the grid = 0.10 s
DFT grid points: 220973 (2630 points/atom)

                      *** Start SCF Iterations ***

 Iter     DIIS Error       Energy change      Electrons        XC Energy           Energy        Time(s)
----------------------------------------------------------------------------------------------------------
Purifying guess...Purification run on GPU (mixed precision)  using TC2 purification.
Density matrix purification finished after a total of 0 matrix multiplies
THRESPDP set to 1.00e+00
                >>> SWITCHING TO GRID 1 <<<
   1     0.2037872348   -2017.7347187380   315.9977416556  -216.7512736380   -2017.7347187380     1.23  
                >>> SWITCHING TO GRID 0 <<<
   2     0.2654210262      +2.9959689702   315.9873030183  -231.0875791470   -2014.7387497678     1.13  
   3     0.2787004633      +1.3787546192   315.9863236247  -227.0139400022   -2013.3599951486     1.09  
   4     0.1933887796      -5.2751474344   315.9866798626  -225.8450958047   -2018.6351425830     1.09  
   5     0.0204442633     -11.4936401361   315.9862731835  -225.2554235360   -2030.1287827191     1.09  
